sdk-setup/src/mer-sdk-chroot

```diff
@@ -75,6 +75,8 @@ if cmp -s /proc/$PPID/mountinfo /proc/self/mountinfo; then
     exit 1
 fi
 
+SFDK_GNUPGHOME_DIR='.gnupg.sailfishsdk'
+
```

Comment:
It should be SDK_, not SFDK_ in this context, and it would be better moved closer to the top, maybe just before MY_SSH_AUTH_SOCK and maybe name it similarly - MY_GNUPGHOME?